app/index.js

```diff
@@ -17,6 +17,8 @@ const createRPC = require('./rpc');
 const notify = require('./notify');
 
 app.commandLine.appendSwitch('js-flags', '--harmony');
+app.commandLine.appendSwitch('enable-transparent-visuals');
+app.commandLine.appendSwitch('disable-gpu');
```

Comment:
@rauchg i don't know seems to be need electron for transparent background.. ref: [Limitations](https://github.com/electron/electron/blob/master/docs/api/frameless-window.md#limitations)